Components/Auth/SocialAuthButton.js

```diff
@@ -28,7 +28,12 @@ const SocialAuthButton = ({ platform }) => {
             },
         })
     }
-
+    
+    var loginText = "Log In With " + platform.friendlyName
+    if (platform.friendlyName == 'Apple') {
```

Comment:
We should key off `id` here and make to that we do a `toLowerCase()` to make sure case doesn't mess with us. Also use `===`